services/api/service.go

```diff
@@ -1727,6 +1727,7 @@ func (api *RelayAPI) checkFloorBidValue(opts bidFloorOpts) (*big.Int, bool) {
 	if err != nil && !errors.Is(err, redis.Nil) {
 		opts.log.WithError(err).Error("failed to get delivered payload slot from redis")
 	} else if opts.submission.BidTrace.Slot <= slotLastPayloadDelivered {
+		opts.simResultC <- &blockSimResult{false, false, nil, nil}
```

Comment:
should we guard this with an env var because it's causing additional simulation load?